public/friendlycode/js/fc/filesystem-sync.js

```diff
@@ -23,6 +23,11 @@ define(["jquery"], function($) {
       request.done(function() {
         if(request.status !== 201 && request.status !== 200) {
           console.error("[Bramble] Server did not persist ", path, ". Server responded with status ", request.status);
+          return;
+        }
+
+        if(context._callback) {
+          context._callback();
```

Comment:
`context` is the scope of `pushFileChange` right? Where might `_callback` be set?